Userland/DevTools/HackStudio/main.cpp

```diff
@@ -185,6 +185,11 @@ void set_current_editor_wrapper(RefPtr<EditorWrapper> wrapper)
     s_hack_studio_widget->set_current_editor_wrapper(wrapper);
 }
 
+void show_file_title_modified()
```

Comment:
Can you choose a better name? This name makes it seem like this function also sets the 'modified state'